download_stanford_data.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
-mkdir Data
+if [ ! -d "Data" ]; then
+	mkdir Data
+fi
```

Comment:
assuming we tested to ensure syntax is right. 
it is cleaner than my just letting fail through approach.

Reply:
I believe I tested and it worked